pkg/backup/util/aws_ebs.go

```diff
@@ -147,10 +148,11 @@ func (e *EC2Session) DeleteSnapshots(snapIDMap map[string]string) error {
 		lock       sync.Mutex
 	)
 
-	eg := new(errgroup.Group)
+	eg, _ := errgroup.WithContext(context.Background())
+	workerPool := NewWorkerPool(e.concurrency, "delete snapshots")
 	for volID := range snapIDMap {
 		snapID := snapIDMap[volID]
-		eg.Go(func() error {
+		workerPool.ApplyOnErrorGroup(eg, func() error {
```

Comment:
What is the batch size. How many snapshots will be deleted concurrently.

Reply:
Its 3 - https://github.com/pingcap/tidb-operator/blob/87b28741cf88bec658ff80745a21d13e0150409d/cmd/backup-manager/app/clean/clean.go#L46